static/src/javascripts/projects/journalism/jtbd-survey.js

```diff
@@ -139,9 +134,7 @@ const init = (): void => {
 
     mediator.emit('journalism:modules:jtbd', true);
 
-    localStorage.setIfNotExists('gu.jtbd.questions', qs, {
-        expires: endOfSurvey,
-    });
+    localStorage.setIfNotExists('gu.jtbd-updated.questions', qs);
```

Comment:
why doesn't it expire at end of survey anymore?

Reply:
the expiry option in the storage module is only a working if there is someone requesting a value _after_ it has expired. in other words, it does not prevent a value to hang in there forever. afaik, the only place where you can specifically address this issue is here:

https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/common.js#L126-L135